app/client-v2/src/stores/projectStore.ts

```diff
@@ -0,0 +1,132 @@
+import { getApiUrl } from "@/config";
```

Comment:
This file isn't hugely readable - I wouldn't be averse to splitting it up into state, getters and actions - or at least putting in a bit of whitespace between sections and methods,

Reply:
good idea il do all this project store comments in the pt2 PR as it will have to restructure in that as well

Reply:
Why not do it here and merge into pt2? Shouldn't be too many conflicts unless you've done major restructuring between pt1 and 2, which seems unlikely. This is partly me being selfish as it will be easier for me to review this way though! 😆

Reply:
sorry i just already did it :cry: ... the merge would've just been nasty to do because of restructuring

Reply:
Ah no worries